osde2e/Dockerfile

```diff
@@ -1,13 +1,13 @@
-FROM registry.ci.openshift.org/openshift/release:golang-1.18 AS builder
 
+# THIS FILE IS GENERATED BY BOILERPLATE. DO NOT EDIT.
+FROM registry.ci.openshift.org/openshift/release:golang-1.18 AS builder
+    
 ENV PKG=/go/src/github.com/openshift/aws-vpce-operator/
-WORKDIR ${PKG}
-
-# compile test binary
+WORKDIR ${PKG}    
 COPY . .
-
+    
```

Comment:
not: trailing whitespace

Reply:
PTAL https://github.com/openshift/boilerplate/pull/272